package.json

```diff
@@ -68,14 +68,14 @@
     "clean": "rm -rf dist && shx mkdir -p dist/popup",
     "watch": "npm-run-all --parallel watch:*",
     "copy:locales": "node pontoon-to-webext",
-    "copy:assets": "shx cp -r src/manifest.json src/icons dist/ && svgo src/icons -o dist/icons && shx cp -r src/popup/*.html src/popup/FiraSans-Regular.* dist/popup/",
+    "copy:assets": "shx cp -r src/manifest.json dist/ && svgo src/icons -o dist/icons --quiet && shx cp -r src/popup/*.html src/popup/FiraSans-Regular.* dist/popup/",
```

Comment:
Added `--quiet` flag, since flooding the console w/ minimization results each time seems unnecessary.